### PR TITLE
Reindent and update Twilio references in Ruby snippets (part 2)

### PR DESCRIPTION
**Pull request by hortega**
- Reindent multiparameter method calls
- Update reference to Twilio console in comments
- Update comments pointing at helper library in Ruby snippets
- Fix double indent on Ruby snippets
- Add new line character to end of all Ruby files that didn't have it
before appling ruby-beautify, as it has a bug affecting files with
no new line character and it is a good practice anyway

### CODE REVIEW

File: rest/voice/outbound-calls/example-4/example-4.4.x.rb [hunk 1]
```diff
@@ -8,10 +8,10 @@
...
-    :method => "GET",
-    :status_callback => "https://www.myapp.com/events",
-    :status_callback_method => "POST",
-    :status_callback_event => ["initiated", "ringing", "answered", "completed"])
+  :to => "+14155551212",
+  :from => "+18668675309",
+  :method => "GET",
+  :status_callback => "https://www.myapp.com/events",
+  :status_callback_method => "POST",
+  :status_callback_event => ["initiated", "ringing", "answered", "completed"])
```
Comment (juancarlospaco) [inline rest/voice/outbound-calls/example-4/example-4.4.x.rb]: Should this lines be a a hanging indent like the `:url` @hortega :grey_question:

Comment (acamino) [inline rest/voice/outbound-calls/example-4/example-4.4.x.rb]: In situations like this you can place the first parameter right below the open paren, e.g.

```ruby
call = @client.account.calls.create(
  :url => "http://demo.twilio.com/docs/voice.xml",
  # ...
```

**Comment by acamino** [comment]
Thanks @hortega! Would you mind address @juancarlospaco's comment please?
Besides that comment - LGTM.

**Comment by hortega** [comment]
Thanks @acamino @juancarlospaco , should be all fixed now

**Pull request merged**